app/views/components/numbered_list.scala.html

```diff
@@ -0,0 +1,26 @@
+@*
```

Comment:
Is this component used?

Reply:
No, it was made to be used but it wouldn't render properly with the link in the list.

Reply:
Resolved